ports/geckolib/gecko_bindings/structs_debug.rs

```diff
@@ -2792,7 +2753,7 @@ impl ::std::clone::Clone for nsIRequest {
 #[repr(C)]
 #[derive(Debug, Copy)]
 pub struct EventStates {
-    pub mStates: ::std::os::raw::c_ulong,
+    pub mStates: ::std::os::raw::c_ulonglong,
```

Comment:
Is it safe to assume `long long` is 64 bits on the systems we care about, regardless of whether we're compiling on 32 or 64 bits?